docs/backstage/pipelines/entries.jsonnet

```diff
@@ -52,18 +52,18 @@
       type_name: 'Custom["BackstageAPI"]',
       source: {
         filter: '$.apiVersion == "backstage.io/v1alpha1" && $.kind == "API"',
-        name: '$.metadata.name',
-        external_id: '$.metadata.namespace + "/" + $.metadata.name',
+        name: '_.get($.metadata, "name", "default API name")',
+        external_id: '_.get($.metadata, "namespace", "default description") + "/" + _.get($.metadata, "name", "default API name")',
         aliases: [
-          '$.metadata.name',
+          '_.get($.badKey, "name", "default alias")',
```

Comment:
did you mean to commit `$badKey`? 

also, in these cases, we're sure that accessing nested field as normal still works? i assume so but wanted to check. We probably don't need all the examples of `_.get()` in the sample docs IMO (lmk what you think). Hopefully it's clear from `expressions.md` that you can use both methods

Reply:
I think that was just testing out aliases. I've replaced it with something less confusing